src/services/sync/repo/user.repo.js

```diff
@@ -38,6 +38,9 @@ const createUser = async (user) => {
 
 const findUsersByEmail = async (email) => userModel.find({ email: email.toLowerCase() }).lean().exec();
 
+const findBySchoolAndName = async (schoolId, firstName, lastName) =>
```

Comment:
```suggestion
const findUserBySchoolAndName = async (schoolId, firstName, lastName) =>
```
other finds describe what they find...